### PR TITLE
:bug:Source Chartmogul: Refactor CustomerCount Stream

### LINKED ISSUE #27953: Source Chartmogul: `Interval` input should follow pattern of one stream per interval
### Topic

Interval pattern

### Revelant information

The `Interval` input field in the Chartmogul source connector currently consists of a dropdown menu with time interval options (ie, day, month, year) for the CustomerCount stream.
 
As per @sherifnada's recommendation, this could instead follow the pattern found in other connectors ([example](https://docs.airbyte.com/integrations/sources/tiktok-marketing/#supported-streams-and-sync-modes)) of having one stream per interval.

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Fixes #27953

## How
*Describe the solution*

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

Yes, there are breaking changes. The interval field was removed from the config, and the customer_count stream was removed and replaced by 4 other streams.

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-chartmogul/source_chartmogul/spec.yaml [hunk 1]
```diff
@@ -6,7 +6,6 @@ connectionSpecification:
   required:
     - api_key
     - start_date
-    - interval
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-chartmogul/source_chartmogul/spec.yaml:9]: look like braking change 

What occurs if a user already has an established connection with a set interval?

**Review by lazebnyi — DISMISSED** [review]
Look like braking change 

also need bump version and change log

File: airbyte-integrations/connectors/source-chartmogul/source_chartmogul/spec.yaml (hunk 1 above)
Comment (topefolorunso) [reply airbyte-integrations/connectors/source-chartmogul/source_chartmogul/spec.yaml:9]: Yes, It's a breaking change.

**Comment by topefolorunso** [comment]
> also need bump version and change log

Done @lazebnyi

**Review by marcosmarxm — COMMENTED** [review]
@lazebnyi can you review the code part? I'll help with the breaking change release and connector update.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**